### PR TITLE
Popup

### PR DESCRIPTION
**Pull request by maaloum**
### In this PR, I :
- Created a folder for Javascript where the pop-up window details are in app.js.
- Created a folder for different CSS styles.
- added, in app.js, the code for rendering different cards by creating an object for data.
- Created the pop-up window using JS, HTML, and CSS.

**Note:  My coding partner was not able to show up since yesterday, that is why I had to work on this project as solo**

### CODE REVIEW

File: javascript/app.js [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+    });
+  }
+}
+
+const projectnBtn = document.querySelectorAll('#popbtn');
+projectnBtn.forEach((btn, ind) => {
+  btn.addEventListener('click', () => {
+    Popup(ind);
+  });
+});
```
Comment (leonardodiasb) [inline javascript/app.js:146]: - [ ] The close button of your popup is not completely removing the popup, resulting in multiple popup windows after opening and closing a project. Kindly fix that.
image

File: CSS/style.css [hunk 2]
```diff
@@ -413,12 +421,87 @@ img:hover {
...
+  padding-top: 25px;
+}
+
+.popup-buttons > .pop-btn {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-size: 15px;
+  font-weight: 500;
+  line-height: 20px;
```
Comment (leonardodiasb) [inline CSS/style.css:506]: - [ ] Your image in the popup looks a little bit stretched and your popup is higher than the screen making it difficult to see all content inside. Also your tags and buttons are a little bit off from the Figma design. Kindly fix the height, paddings and margins of your elements.
Your mobile design:
image

File: CSS/media.css [hunk 3]
```diff
@@ -90,4 +88,53 @@
...
+  }
+
+  .right-side > .project-languages {
+    margin-bottom: 20px;
+  }
+
+  .popup-bottom {
+    display: flex;
+    margin-bottom: 40px;
+  }
```
Comment (leonardodiasb) [inline CSS/media.css:139]: - [ ] Your popup is not centered in some screen sizes and the image, buttons and tags are a little bit off from the Figma design. Kindly fix the heights, margins, paddings, etc...
Your desktop design:
image

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @maaloum, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
...
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your README and PR message are descriptive ✔️

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: javascript/app.js (hunk 1 above)
Comment (maaloum) [reply javascript/app.js:146]: Kindly, any suggestion on how to fix that?

File: CSS/media.css [hunk 4]
```diff
@@ -90,4 +88,46 @@
...
+    border: 1px solid #ebecf0;
+    box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+    border-radius: 16px;
+  }
+
+  .popimg {
+    width: 100%;
+    height: 300px;
+    margin-bottom: 20px;
+  }
```
Comment (mukhammadrizooff) [inline CSS/media.css:110]: - [ ] [REQUIRED] Good job so far! There are design issues with your Popup. Please, in desktop and mobile versions, your popup does not open on full screen. See the problem below:

- ### [HINT] Please watch [this tutorial](https://youtu.be/MBaw_6cPmAw)!

- Desktop version.

image

- Mobile version.

image

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @maaloum, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
- No linters ✔️ 
- Good implementations ✔️ 
- Popup is working well ✔️ 
 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: CSS/style.css [hunk 5]
```diff
@@ -413,12 +421,84 @@ img:hover {
...
+}
+
+.popimg {
+  position: relative;
+  width: 100%;
+  margin-bottom: 10px;
+  border-radius: 8px;
+}
+
+.popup-buttons {
```
Comment (isaacpitwa) [inline CSS/style.css:489]: - [ ][OPTIONAL] In mobile view, the pop-up seems to be overflowed horizontally. I think this issue is caused by the fixed width assigned to the pop-up buttons. I think using relative measurements  for the width could help solve this issue
Screenshot 2022-07-27 at 11 53 06

File: CSS/style.css [hunk 6]
```diff
@@ -413,12 +421,84 @@ img:hover {
...
+  overflow: auto;
+  background: rgb(193 199 208 / 50%);
+  backdrop-filter: blur(3px);
+  padding: 50px 16px;
+}
+
+.pop-content {
+  width: 100%;
+  margin: 0 auto;
+  height: 100vh;
```
Comment (isaacpitwa) [inline CSS/style.css:448]: Great job making the requested changes 👏🏽

- [ ] [OPTIONAL] the changes made altered could have deisgn to a full viewport pop-up height. I
Screenshot 2022-07-27 at 11 29 22
 think adjusting the height to default in **desktop view** could help solve this problem.

**Review by isaacpitwa — APPROVED** [review]
# Approved 💯 ✅ 🥳 🎈 🎉 

Hi @maaloum  👋🏻 

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 



## Highlights ✅ 
Highly organized and clean code ✔️ 👨🏻‍💻 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 


## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Check the comments under the review.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**